### ISSUE TITLE
Use Fancytree for autocomplete menu in keyboard paste dialog

### ISSUE THREAD

**Issue by aecreations** [aecreations/clippings#519]
The autocomplete menu in the keyboard paste dialog, when in "search by name" mode, is a custom, hand-built UI component. There is a minor UI bug with keyboard navigation when using the up-arrow key, but the biggest flaw is the lack of accessibility - screen readers aren't able to handle keyboard navigation.

The clipping search result that is populated in the autocomplete menu should be rendered using Fancytree, which has better keyboard navigation and fully supports accessibility tools such as screen readers.

**Comment by aecreations** [comment]
Still doesn't work. This may need to be an accepted limitation to avoid changing existing keyboard navigation functionality.
